embrace-android-core/src/main/kotlin/io/embrace/android/embracesdk/internal/envelope/log/LogEnvelopeSource.kt

```diff
@@ -1,10 +1,11 @@
 package io.embrace.android.embracesdk.internal.envelope.log
 
+import io.embrace.android.embracesdk.internal.logs.LogRequest
 import io.embrace.android.embracesdk.internal.payload.Envelope
 import io.embrace.android.embracesdk.internal.payload.LogPayload
 
 public interface LogEnvelopeSource {
     public fun getBatchedLogEnvelope(): Envelope<LogPayload>
 
-    public fun getNonbatchedEnvelope(): List<Envelope<LogPayload>>
+    public fun getNonbatchedEnvelope(): List<LogRequest<Envelope<LogPayload>>>
```

Comment:
no need to change it now as it's not part of your PR but I don't like "nonbatched" lol

Unbatched is an English word right?

Reply:
Lol I actually changed it to something else but it sounded weird so I just reverted it. Let me take another stab at it. Do you think `SingleLogBatch` is better?

Reply:
Did a pass on the names and got rid of any "nonbatch" references. I kept one "unbatched", but the rest use `SingleLog` or something that denotes that.